@here/harp-examples/src/datasource_object-picking.ts

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MapControls, MapControlsUI } from "@here/harp-map-controls";
+import { CopyrightElementHandler, MapView, PickResult } from "@here/harp-mapview";
+import { APIFormat, OmvDataSource } from "@here/harp-omv-datasource";
+import { accessToken } from "../config";
+
+/**
+ * This example showcases how picking works.
+ *
+ * For enable line picking set `enableRoadPicking: true` in [[MapViewOptions]] and set
+ * `createTileInfo: true` in [[OmvWithRestClientParams|OmvWithCustomDataProvider]].
+ * For enable polygon picking set `gatherFeatureIds: true` in
+ * [[OmvWithRestClientParams|OmvWithCustomDataProvider]].
+ * For enable text element picking set `gatherFeatureIds: true` in
+ * [[OmvWithRestClientParams|OmvWithCustomDataProvider]].
+ *
+ * Now, let's write an event that fires when the user clicks the map canvas:
+ * ```typescript
+ * [[include:datasource_object_picking_1.ts]]
+ * ```
+ *
+ * All the data handling is covered by the `handlePick` function. Here we find the
+ * intersected objects, pick the first one in the array and display its
+ * data inside the helper box
+ * ```typescript
+ * [[include:datasource_object_picking_2.ts]]
+ * ```
+ */
+
+export namespace GeoJsonPickingExample {
```

Comment:
Maybe rename the namespace? This example doesn't have anything to do with GeoJson, and this is how it'll be displayed inside the documentation.

Reply:
Done